personal/privacyidea/class_PrivacyIdeaUtils.inc

```diff
@@ -136,6 +134,10 @@ class PrivacyIdeaUtils implements PILog
      */
     public function countUserTokens($tokens)
     {
+        if (!$this->authenticate()) {
+            return;
+        }
```

Comment:
What will happen if we return from here. Is there already some auth-failure handling (REST API auth against privacyIDEA failing, I mean).

@dzatoah cc

Reply:
This method is public and there is a TODO to modify this so it fetches tokens directly from PrivacyIDEA without passing them in. However it is currently only used internally and needs to be passed tokens in, so the check can be left out for now until that is actually implemented.